libs/langchain-google-genai/src/chat_models.ts

```diff
@@ -198,6 +207,17 @@ export class ChatGoogleGenerativeAI
       throw new Error("`topK` must be a positive integer");
     }
 
+    this.candidateCount = fields?.candidateCount ?? this.candidateCount;
+    if (this.candidateCount && this.candidateCount <= 0) {
+      throw new Error("`candidateCount` must be above 1.");
+    }
+
+    this.stopSequences = fields?.stopSequences ?? this.stopSequences;
```

Comment:
The type of the stopSequences is Array of strings, I'm not sure we need the next two lines.